website/templates/emails/email_contributors_of_public_registrations.txt.mako

```diff
@@ -0,0 +1,12 @@
+Hello ${user.fullname},
+
+Sara and Brian N., please revise below.
```

Comment:
We have a longer email explaining what is happening, right?  Is this a follow-up?